Gem/Code/Source/Components/UI/UiMatchPlayerCoinCountsComponent.cpp

```diff
@@ -109,13 +109,23 @@ namespace MultiplayerSample
         {
             if (const PlayerIdentityComponent* identity = playerHandle.GetEntity()->FindComponent<PlayerIdentityComponent>())
             {
-                return identity->GetPlayerName();
+                PlayerNameString playerName = identity->GetPlayerName();
+                if (playerName.empty())
+                {
+                    if(const auto playerDebugId= playerHandle.GetEntity()->FindComponent<Multiplayer::NetworkDebugPlayerIdComponent>())
+                    {
+                        AZStd::string playerIdText = AZStd::string::format("<debug id: %i>", static_cast<uint32_t>(playerDebugId->GetPlayerId()));
```

Comment:
Nit: I believe this should be %u for a uint32_t

Reply:
Perfect. Thanks!